apogee/tools/__init__.py

```diff
@@ -105,6 +176,31 @@ def atomic_number(elem):
     except (NameError,KeyError):
         return elements.__dict__[elem.lower().capitalize()].number
 
+def sigma_XH(elem,Teff=4500.,M_H=0.,SNR=100.,dr=None):
+    """
+    NAME:
+       sigma_XH
+    PURPOSE:
+       return uncertainty in a given element at specified effective 
+       temperature, metallicity and signal to noise ratio
```

Comment:
Maybe add a reference to Holtzman et al. here or to the website where these coefficients are given? Just so we remember where they came from.